manifests/y/yt-dlp/yt-dlp/nightly/2024.07.03.232825/yt-dlp.yt-dlp.nightly.locale.en-US.yaml

```
@@ -0,0 +1,88 @@
+
+PackageIdentifier: yt-dlp.yt-dlp.nightly
+PackageVersion: 2024.07.03.232825
+PackageLocale: en-US
+Publisher: yt-dlp
+PublisherUrl: https://github.com/yt-dlp
+PublisherSupportUrl: https://github.com/yt-dlp/yt-dlp/issues
+PackageName: yt-dlp-nightly
+PackageUrl: https://github.com/yt-dlp/yt-dlp-nightly-builds
+License: Unlicense
+LicenseUrl: https://github.com/yt-dlp/yt-dlp/blob/master/LICENSE
+Copyright: pukkandan.ytdlp@gmail.com | UNLICENSE
+ShortDescription: A feature-rich command-line audio/video downloader.
+Description: |-
+  The yt-dlp nightly channel builds.
+  yt-dlp is a feature-rich command-line audio/video downloader with support for thousands of sites.
+  The project is a fork of youtube-dl based on the now inactive youtube-dlc.
+Moniker: yt-dlp-nightly
+Tags:
+- audio
+- audio-downloader
+- download
+- downloader
+- media
+- media-downloader
+- nightly
+- video
+- video-downloader
+- youtube
+- youtube-dl
+- youtube-dlp
+- ytdlp
+ReleaseNotes: |-
+  Core changes
+  - cookies: Fix --cookies-from-browser DE detection on Linux (#10237) by peisenwang
+
+  Extractor changes
+  - brightcove: Upgrade requests to HTTPS (#10202) by bashonly
+  - cloudflarestream: Fix _VALID_URL and embed extraction (#10215) by bashonly
+  - facebook: reel: Fix extraction (#10232) by bashonly
+  - francetv
+    - Detect and raise errors for DRM (#10165) by bashonly
+    - Fix extractor (#10177) by bashonly
+  - generic: Add key_query extractor-arg by bashonly
+  - khanacademy: Fix extractors (#9136) by c-basalt
+  - matchtv: Fix extractor (#10190) by megumintyan
+  - nhkradiru: Fix extractor (#10106) by garret1317
+  - orf: on: Support segmented episodes (#10053) by seproDev
+  - patreoncampaign: Fix campaign_id extraction (#10070) by bashonly
+  - podbayfm: Fix extraction (#10195) by bashonly, seproDev
+  - rtvslo.si: show: Add extractor (#8418) by JSubelj, seproDev
+  - soundcloud: Fix download format extraction (#10125) by bashonly
+  - sproutvideo: Add extractors (#10098) by bashonly, TheZ3ro
+  - tiktok
+    - Detect and raise when login is required (#10124) by bashonly
+    - Fix API extraction (#10216) by bashonly
+  - tubitv
+    - Fix extractor (#9975) by chilinux
+    - series: Fix extractor (#10116) by bashonly
+  - youtube
+    - Extract all formats from multi-language m3u8s (#9875) by bashonly, clienthax
+    - Skip formats if nsig decoding fails (#10223) by bashonly
+    - tab: Fix channel metadata extraction (#10071) by bashonly, shoxie007
+
+  Downloader changes
+  - hls: Apply extra_param_to_key_url from info dict by bashonly
+
+  Networking changes
+  - Request Handler: requests: Bump minimum requests version to 2.32.2 (#10079) by bashonly
+
+  Misc. changes
+  - build
+    - Bump Pyinstaller to >=6.7.0 for all builds (#10069) by bashonly, seproDev
+    - Cache dependencies for macos job (#10088) by bashonly
+    - Use macos-12 image for yt-dlp_macos (#10063) by bashonly
+  - cleanup: Add more ruff rules (#10149) by seproDev
```

Update `ReleaseNotes`

```suggestion
ReleaseNotes: |-
  Core changes
  - Fix --ignore-no-formats-error (#10345) by Grub4K

  Extractor changes
  - douyutv: Do not use dangerous javascript source/URL (#10347) by LeSuisse
```